### PR TITLE
EWL 3269 - Subcategory topics with description

### PR DESCRIPTION
**Pull request by martensc**
Ticket: https://issues.ama-assn.org/browse/EWL-3269

Created subcategory topics with description component

To test:
- Go to Molecules > Components > Subcategory topics with Description
- Check component at all viewports to ensure it doesn't break
- Check hover-states of links

### CODE REVIEW

File: styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/subcategory-topics-description.twig [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<div class="subcategory-topics-description layout">
+  <div class="subcategory-topics-description_wrap">
+    <div class="subcategory-topics-description_title">
+      <div class="subcategory-topics-description_title-inner">
+        {% include 'atoms-eyebrow' %}
+        <a href="{{ url }}" class="subcategory-topics-description_link">Get Started</a>
+      </div>
+    </div>
+    <div class="subcategory-topics-description_teaser">
+      {% include 'atoms-paragraph' with { 'content': 'Nulla vitae elit libero, a pharetra augue. Donec id elit non mi a porta gravida at eget metus. Cras justo odio, dapibus ac facilisis in, egestas eget quam.', 'class': 'b4 category-highlight_teaser' } %}
```
Comment (aCyborg) [inline styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/subcategory-topics-description.twig]: Same thing about the b4 here to use the %type-smaller extend

**Comment by froboy** [comment]
@megansabina this seems related to the previous request.

File: styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/subcategory-topics-description.twig [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<div class="subcategory-topics-description layout">
+  <div class="subcategory-topics-description_wrap">
+    <div class="subcategory-topics-description_title">
+      <div class="subcategory-topics-description_title-inner">
+        {% include 'atoms-eyebrow' %}
+        <a href="{{ url }}" class="subcategory-topics-description_link">Get Started</a>
+      </div>
+    </div>
+    <div class="subcategory-topics-description_teaser">
+      {% include 'atoms-paragraph' with { 'content': 'Nulla vitae elit libero, a pharetra augue. Donec id elit non mi a porta gravida at eget metus. Cras justo odio, dapibus ac facilisis in, egestas eget quam.', 'class': 'type-smaller category-highlight_teaser' } %}
```
Comment (froboy) [inline styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/subcategory-topics-description.twig]: I don't think this needs `type-smaller`. On both http://ama-assn.local/education and the designs it looks like it's the same size as the link text. Please remove `type-smaller`.

File: styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/_subcategory-topics-description.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  @include breakpoint($bp-med) {
+    margin-bottom: 60px;
+  }
+}
+
+// Subcategory title and teaser wrap
+.subcategory-topics-description_wrap {
+  @include breakpoint($bp-med) {
+    @include grid__unit--cols(6);
+    padding-right: 15px;
```
Comment (froboy) [inline styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/_subcategory-topics-description.scss]: This padding seems a little small (on tablet and desktop widths)... the design doesn't specify, but this seems a little tighter than the implementation on production. Maybe bump it up to 40px, that's a common gutter, right?
pattern_lab_-_molecules-subcategory-topics-description

Comment (megansabina) [inline styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/_subcategory-topics-description.scss]: All set!

File: styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/subcategory-topics-description.twig (hunk 2 above)
Comment (megansabina) [inline styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/subcategory-topics-description.twig]: All set!

**Comment by froboy** [comment]
Looks great. Thanks everyone!

**Pull request merged**